modules/osdk-cli-reference-generate-packagemanifests.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-packagemanifests_{context}"]
+= packagemanifests
+
+Running `generate packagemanifests` subcommand is the first step to publishing your operator to a catalog and/or deploying
+it with OLM. This command generates a set of manifests in a versioned directory and a package manifest file for
+your operator. You must run `generate kustomize manifests` first to regenerate kustomize bases
+consumed by this command.
+
+
+.`generate packagemanifests` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`--channel` (string)
+|The channel name for the generated package.
+
+|`--crds-dir` (string)
+|The root directory for Custom Resource Definition (CRD) manifests.
+
+|`--default-channel`
+|Use the channel passed to `--channel` as the default channel of package manifest file.
+
+|`--deploy-dir` (string)
+|The root directory for operator manifests such as deployments and RBAC, for example, `deploy`. This directory is different from that passed to `--input-dir`.
+
+|`--from-version` (string)
+|The semantic version of the operator, from which it is being upgraded.
+
+|`-h`, `--help`
+|Help for `generate kustomize manifests`.
+
+|`--input-dir` (string)
+|The directory to read existing package manifests from. This directory is the parent of individual versioned package directories, and different from `--deploy-dir`.
+
+|`--kustomize-dir` (string)
+|The directory containing kustomize bases and a `kustomization.yaml` for operator-framework manifests. Default: `config/manifests`.
```

Comment:
In general text, please initial cap Kustomize.

I'm inclined to think that `operator-framework` should be treated as inline code.  I'm not familiar with this information, so I'm a little unsure.  Up to you.